contrib/for-tests/volumes/gluster/glusterd.vol

```diff
@@ -0,0 +1,10 @@
+volume management
+    type mgmt/glusterd
+    option working-directory /var/lib/glusterd
+    option transport-type socket,rdma
+    option transport.socket.keepalive-time 10
+    option transport.socket.keepalive-interval 2
+    option transport.socket.read-fail-log off
+#   option base-port 49152
```

Comment:
Except "rpc-auth-allow-insecure on" it's just default glusterd.vol, including commented out "base-port" option. I'll add some note there.